ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -18,12 +18,12 @@
         @passObject={{true}}
         @objectKeys={{array "type" "version"}}
         class="is-marginless"
-        data-test-secrets-engines-select
+        data-test-search-select="secrets-engines"
```

Comment:
I've also seen us leverage the test selector in the search select component with the `id` so here that'd be `[data-test-component="search-select"]#secret-engines-select`

No need to change, more of an FYI

Reply:
good suggestion! will keep that in mind next time!